src/kubernetes_cluster/spec/message.rs

```diff
@@ -168,6 +168,16 @@ impl MessageContent {
     {
         self.get_APIResponse_1()
     }
+
+    pub open spec fn get_msg_id(self) -> nat
```

Comment:
I would suggest you do pattern matching here in case we later add more message types without chan id.